web/clusters.go

```diff
@@ -296,7 +295,7 @@ type ClustersRow struct {
 
 type ClustersTable []*ClustersRow
 
-func NewClustersTable(s services.ChecksService, client consul.Client, clusters map[string]*cluster.Cluster) (ClustersTable, error) {
+func NewClustersTable(s services.ChecksService, t services.TagsService, client consul.Client, clusters map[string]*cluster.Cluster) (ClustersTable, error) {
```

Comment:
Is the consul client parameter used here for anything?

Reply:
leftover, thx